frontend/src/styles/home/FeaturedProjectCard.module.css

```diff
@@ -1,33 +1,19 @@
-
 #cardContainer {
-  margin-top: 0px;
-  margin-bottom: 3vw;
-  display: flex;
+  flex-direction: column;

```

Comment:
i believe these first three rules aren't doing anything because the default display is block?
![image](https://user-images.githubusercontent.com/45301066/183259560-0ead1ae8-ebf0-439c-a6f9-d2fc59129461.png)

Reply:
Apologies, there might be some redundant styling still left over that I forgot to clean up

Reply:
@heylookitsme What do you mean by those first three rules? The ones that your comment indicates are already deleted??